.gitignore

```diff
@@ -5,5 +5,5 @@
 /captures
 .externalNativeBuild
 .cxx
-local.properties
+key.properties
 key.properties
```

Comment:
local.properties must be ignored (key was already ignored and now it's redundant)